test/e2e/fixtures.go

```diff
@@ -36,6 +36,18 @@ func skipIfNotBenchmarkTest(tb testing.TB) {
 	}
 }
 
+func skipIfNotAntreaIPAMTest(tb testing.TB) {
+	if !testOptions.enableAntreaIPAM {
+		tb.Skipf("Skipping AntreaIPAM test: %s", tb.Name())
+	}
+}
+
+func skipIfAntreaIPAMTest(tb testing.TB) {
+	if !testOptions.enableAntreaIPAM {
```

Comment:
Here skip if antrea ipam is enabled, so `if testOptions.enableAntreaIPAM`?

Reply:
Thanks. Fixed.